tools/oca_projects.py

```diff
@@ -10,6 +10,8 @@
 
 """
 
+from github_login import login
```

Comment:
Why? `ImportError` is good I think. I would handle specially only error messages which are misleading.